src/Pim/Bundle/CatalogBundle/Doctrine/Common/Saver/ProductSaver.php

```diff
@@ -115,4 +116,49 @@ protected function validateProduct($product)
             );
         }
     }
+
+    /**
+     * Drops the current completenesses from the database for a list of products.
+     *
+     * @param ProductInterface[] $products
+     */
+    protected function dropCompletenesses(array $products)
```

Comment:
My intuition is that this, should be done somewhere else 'completenessRemover' something like that. you don't *have* to do it in this PR. But i'd like to know your take.